### ISSUE TITLE
uvloop does not implement _write_to_self

### ISSUE THREAD

**Issue by atbentley** [MagicStack/uvloop#66]
* **uvloop version**: 0.7.0
* **python version**: 3.6.0
* **platform**: OS X El Capitan 10.11.6

With asyncio default event loop the following works fine:
```
import asyncio

async def async_gen():
    yield 5

async def run():
    val = await async_gen().__anext__()
    print(val)

asyncio.get_event_loop().run_until_complete(run())
```

With uvloop, the following will raise a `AttributeError: 'Loop' object has no attribute '_write_to_self'`
```
import asyncio
import uvloop
asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())

async def async_gen():
    yield 5

async def run():
    val = await async_gen().__anext__()
    print(val)

asyncio.get_event_loop().run_until_complete(run())
```

After a quick glance it looks like some of the new async generator code is calling `_write_to_self`, in particular the `_asyncgen_finalizer_hook` method. However it doesn't seem like the uvloop event loop has implemented that method yet.

**Comment by 1st1** [comment]
Thanks for reporting this. We'll fix this in the next release (soon)

**Comment by 1st1** [comment]
Fixed in 0.7.1. Thanks for reporting this!

**Comment by atbentley** [comment]
Thanks for the quick fix, Yury!
